#! /usr/bin/env python

PACKAGE='sensors'
import roslib
roslib.load_manifest(PACKAGE)

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()
#       Name       Type      Level Description                Default Min     Max
gen.add("numDisparities", int_t,    0,    "Number of Disparity",   3,      1,     5)
gen.add("blockSize",      int_t,    0,    "Block Size",           12,      0,   125)
gen.add("maxDepth",    double_t,    0,    "Max Depth",          30.0,      0, 100.0)

exit(gen.generate(PACKAGE, "sensors", "StereoVisionParams"))
